tox.ini

```diff
@@ -50,7 +50,7 @@ extras =
 commands =
     sunpy: pytest -vvv -s -ra --pyargs sunpy.net.jsoc --remote-data=any {posargs}
     !online: {env:PYTEST_COMMAND} {posargs}
-    online: {env:PYTEST_COMMAND} --reruns 2 --timeout=180 --remote-data=any --email nabil.freij@gmail.com {posargs}
+    online: {env:PYTEST_COMMAND} --reruns 2 --reruns-delay 30 --timeout=180 --remote-data=any --email nabil.freij@gmail.com {posargs}
```

Comment:
Dont you want to use the test email here?

Reply:
Woops.